numpydoc/numpydoc.py

```diff
@@ -184,6 +183,33 @@ class NumpyCDomain(ManglingDomainBase, CDomain):
     }
 
 
+def match_items(lines, content_old):
+    """Create the right items for lines.
```

Comment:
Actually, this has nothing to do with docutils. The ``mangle_docstring`` function changes the lines but not the items. That's pretty much a design error. It should keep track of both.